File: C#/백준/Bronze/17598. Animal King Election/Animal King Election.cs
using System;

namespace q{
    class w{
        static void Main(){
            int cnt=0;
            for (int i=0; i<9; i++){
                if(Console.ReadLine()=="Lion"){
                    cnt++;
                }
            }
            Console.WriteLine((cnt>=5)?"Lion":"Tiger");
        }
    }
}

File: Java/백준/Bronze/17598. Animal King Election/Animal King Election.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int cnt = 0;
        String l = "Lion";
        for (int i=0; i<9; i++){
            String s = in.next();
            if(s.equals(l)){
                cnt++;
            }
        }
        System.out.println((cnt>=5)?"Lion":"Tiger");
    }
}